app/src/main/res/values/strings.xml

```diff
@@ -124,8 +122,8 @@
     <string name="attack">Attack</string>
     <string name="defense">Defense</string>
     <string name="stamina_hp">HP</string>
-    <string name="missing_inputs">Missing data. Please fill CP and HP inputs.</string>
-    <string name="ivtext_no_possibilities">There are no IV possible combinations for this input. Please double check the information.</string>
+    <string name="missing_inputs">CP and HP are required.</string>
+    <string name="ivtext_no_possibilities">There are no possible combinations. Make sure the values you have entered are correct.</string>
```

Comment:
In my opinion (of course):

Compared to "Missing data.", "CP and HP are required" is more to the point and at the same time implies that the user will need to fill them in.

"IV" in our case is implied as the app is an IV calculator. So I removed that.

Compared to "for this input. Please double check the information.", "Make sure the values you have entered are correct." is more concise and clear.